packages/cli-platform-ios/src/tools/runExplicitMode.ts

```diff
@@ -0,0 +1,42 @@
+import {logger} from '@react-native-community/cli-tools';
+import chalk from 'chalk';
+import {getProjectInfo} from './getProjectInfo';
+import {
+  promptForConfigurationSelection,
+  promptForSchemeSelection,
+} from './prompts';
+
+interface Args {
+  scheme?: string;
+  mode?: string;
+}
+
+export async function runExplicitMode(args: Args): Promise<Args> {
```

Comment:
I would rename this function since it's not actually doing any `run` related stuff. It's just returning scheme and mode right?